test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
I'm sorry I wasn't clear before.

I agree with @tkyi and @d2lam that we should be striving to use real data in our tests.  I agree with @FenrirUnbound that this discussion is over something trivial.

But this should not have taken 4+ hours to discuss this.  We will use the `scmUri` I provided for the test.